client/.gitignore

```diff
@@ -1,4 +1,5 @@
 .eslintrc.js
 .vscode/
 
-node_modules/
\ No newline at end of file
+node_modules/
+package-lock.json
```

Comment:
Generally actually don't need to gitignore package-lock.json (https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5)
It's also used by bots to determine if you have vulnerabilities, so it's good to keep

Reply:
ok! updating now.